lms/djangoapps/mobile_api/video_outlines/serializers.py

```diff
@@ -209,6 +209,10 @@ def video_summary(course, course_id, video_descriptor, request, local_cache):
         for lang in transcript_langs
     }
 
+    if video_descriptor.only_on_web:
+        video_url = None
+        transcripts = {}
```

Comment:
This will conflict with https://github.com/edx/edx-platform/pull/7432.  I suggest moving this logic up above (where the `video_url` is computed) once this code is merged with PR-7432.